services/minespace-web/src/components/Forms/projects/projectSummary/DocumentUpload.tsx

```diff
@@ -72,12 +62,25 @@ export const DocumentUpload: FC<DocumentUploadProps> = (props) => {
     ...IMAGE,
   };
 
+  useEffect(() => {
+    props.change(FORM.ADD_EDIT_PROJECT_SUMMARY, "spatial_documents", []);
+    props.change(FORM.ADD_EDIT_PROJECT_SUMMARY, "support_documents", []);
+  }, []);
```

Comment:
just checking- do the fields populate properly when you view an existing record (with existing spatial/support documents)?

Reply:
Yup the fields show up when someone has uploaded a spatial or support document in the file uploader.